fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/service/LoanAccrualsProcessingServiceImpl.java

```diff
@@ -427,9 +427,9 @@ private void addInterestAccrual(@NotNull Loan loan, @NotNull LocalDate tillDate,
         AccrualPeriodData period = accrualPeriods.getPeriodByInstallmentNumber(installment.getInstallmentNumber());
         MonetaryCurrency currency = accrualPeriods.getCurrency();
         Money interest = null;
-        boolean isFullPeriod = isFullPeriod(tillDate, installment);
+        boolean isClosedPeriod = isAfterPeriod(tillDate, installment);
```

Comment:
Something like "pastDue" or "pastPeriod" field name might be better...

Reply:
Renamed it.